src/qt/dash.cpp

```diff
@@ -693,7 +690,12 @@ int main(int argc, char *argv[])
     app.parameterSetup();
     // Load GUI settings from QSettings
     app.createOptionsModel(gArgs.GetBoolArg("-resetguisettings", false));
-
+    // Load custom application fonts and setup font management
+    if (!GUIUtil::loadFonts()) {
+            QMessageBox::critical(0, QObject::tr(PACKAGE_NAME),
+                                  QObject::tr("Error: Failed to load application fonts."));
+            return EXIT_FAILURE;
```

Comment:
please fix formatting (there's an extra tab here)
```suggestion
        QMessageBox::critical(0, QObject::tr(PACKAGE_NAME),
                              QObject::tr("Error: Failed to load application fonts."));
        return EXIT_FAILURE;
```